### PR TITLE
fix: adding a query parameter in Playground requires one extra click to type in search field

### PR DESCRIPTION
**Pull request by MrFlashAccount**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Description of Changes Made (if issue reference is not provided)**

this PR add focusing in input on opening the menu and on typing within the list

### CODE REVIEW

File: packages/cubejs-playground/src/QueryBuilder/MemberDropdown.tsx [hunk 1]
```diff
@@ -117,8 +118,25 @@ export default function MemberMenu({
...
+      onClick={() => {
+        // we need to delay focusing since React needs to render <Menu /> first :)
+        setTimeout(() => {
+          searchInputRef.current?.focus();
+        });
+      }}
       overlay={
         <Menu
+          onKeyDown={(e) => {
+            if (['ArrowDown', 'ArrowUp'].includes(e.key)) {
```
Comment (tenphi) [inline packages/cubejs-playground/src/QueryBuilder/MemberDropdown.tsx:134]: Looks weird. We do focus on every key except these two? It should break keyboard navigation hard.

Comment (MrFlashAccount) [reply packages/cubejs-playground/src/QueryBuilder/MemberDropdown.tsx:134]: I'm not sure it's weird, though. When we navigate the menu and can't find the needed item, we can start typing, and even if the input isn't focused, the text will be typed in the input. This is a nice pattern :)

https://user-images.githubusercontent.com/61194245/161109725-f9780a93-7382-4f79-bf83-8532e0e68cfd.mov

Comment (MrFlashAccount) [reply packages/cubejs-playground/src/QueryBuilder/MemberDropdown.tsx:134]: Although, there is actually no keyboard navigation except navigation within the list. I've tested.

**Comment by MrFlashAccount** [comment]
I've found a bug, it doesn't work with filters.

File: packages/cubejs-playground/src/QueryBuilder/MemberDropdown.tsx (hunk 1 above)
Comment (tenphi) [reply packages/cubejs-playground/src/QueryBuilder/MemberDropdown.tsx:134]: The pattern is good. But keyboard navigation includes: Space, Enter, PageUp, PageDown, Up, Down, Right, Left (yeah, horizontal, even in vertical layout). Maybe we should test an event for a character and only in that case do the focus.

**Review by tenphi — APPROVED** [review]

**Pull request merged**